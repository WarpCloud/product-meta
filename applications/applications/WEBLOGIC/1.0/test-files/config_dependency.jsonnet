{
  application_type: "WEBLOGIC",
  application_name: "weblogic1",
  application_version: "1.0"
}
