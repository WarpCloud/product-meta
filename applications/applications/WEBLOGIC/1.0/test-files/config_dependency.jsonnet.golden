{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "weblogic1",
            "application_type": "WEBLOGIC"
         },
         "configs": {
            "admin_cpu_limit": 1,
            "admin_cpu_request": 0.10000000000000001,
            "admin_memory_limit": 4,
            "admin_memory_request": 1,
            "admin_storage_limit": 10,
            "ms_cpu_limit": 1,
            "ms_cpu_request": 0.10000000000000001,
            "ms_memory_limit": 4,
            "ms_memory_request": 1,
            "ms_storage_limit": 20
         },
         "instance_settings": { },
         "moduleName": "weblogic",
         "name": "weblogic1-weblogic",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
